### PR TITLE
Adding unit test to twitter project

### PR DESCRIPTION
**Pull request by vglalitha**

### CODE REVIEW

File: .idea/maven.demo.iml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (PoojaMohite-Khoros) [inline .idea/maven.demo.iml:1]: Please align this file

**Review by PoojaMohite-Khoros — CHANGES_REQUESTED** [review]

File: dropwizard-demo/src/main/java/com/dep/resource/DropResource.java [hunk 2]
```diff
@@ -21,42 +21,40 @@
 @Path("/api/1.0/twitter")
 public class Controller {
 
+    @GET
+    @Path("/getTweets")
+    public static ArrayList<String> Gettweets() throws TwitterException {
```
Comment (PoojaMohite-Khoros) [inline dropwizard-demo/src/main/java/com/dep/resource/DropResource.java:26]: GetTweet doesn't naming following standard
Please change to getTweets

**Review by PoojaMohite-Khoros — CHANGES_REQUESTED** [review]

File: .idea/maven.demo.iml (hunk 1 above)
Comment (vglalitha) [reply .idea/maven.demo.iml]: Addressed

File: dropwizard-demo/src/main/java/com/dep/resource/DropResource.java (hunk 2 above)
Comment (vglalitha) [reply dropwizard-demo/src/main/java/com/dep/resource/DropResource.java]: Addressed

**Pull request closed without merging**